e2e/scientists_queries.yaml

```diff
@@ -2,75 +2,83 @@
 name: scientists
 queries:
 
+  # TODO<joka> reintroduce order by DESC(SCORE()) as soon as SCORE is
+  #  reimplemented
   - query: relativ-star-scientists
     type: text
     sparql: |
-      SELECT ?x SCORE(?t) TEXT(?t) WHERE {
+      SELECT ?x TEXT(?t) WHERE {
           ?x <is-a> <Scientist> .
           ?t ql:contains-entity ?x .
           ?t ql:contains-word "relati*"
       }
-      ORDER BY DESC(SCORE(?t))
+    #      ORDER BY DESC(SCORE(?t))
     checks:
-      - num_cols: 3
+      - num_cols: 2
       - num_rows: 1653
-      - selected: ["?x", "SCORE(?t)", "TEXT(?t)"]
+      - selected: [ "?x", "TEXT(?t)" ]
       - contains_row:
-        - "<Albert_Einstein>"
-        - 169
-        - "He realized, however, that the principle of relativity could also be extended
+          - "<Albert_Einstein>"
+          - 169
```

Comment:
If you wonder why the e2e tests all fail:
You removed the score column, so you also have to remove it in all the `contains_row` tests:
(In this case the second column with the `169` value, similar for all of the tests in the queries where we removed the `SCORE()`.